asio/cmake/install-rules.cmake

```diff
@@ -0,0 +1,38 @@
+if(PROJECT_IS_TOP_LEVEL)
+  set(CMAKE_INSTALL_INCLUDEDIR include CACHE PATH "")
```

Comment:
This was wrong `include/asio` generated and is in wrong scope!

Reply:
This is explained by the pinned issue and the comments within: https://github.com/friendlyanon/cmake-init/issues/43

If you don't wish to protect the user from that by default, then this bit can be removed entirely.